action.yml

```diff
@@ -39,6 +39,7 @@ runs:
     - name: Print debug info
       run: |
         conda-lock --version
+        mamba --version
```

Comment:
Yeah, `mamba` is not installed with `pip install conda-lock` (duh), see https://github.com/weiji14/conda-lock-refresh/actions/runs/6270249768/job/17027922366?pr=4#step:3:237:

```
/home/runner/work/_temp/08e949be-5a4c-4221-a19d-f4a0bca225b3.sh: line 2: mamba: command not found
```